UI/src/view/point_asset/pointAssetForm.js

```diff
@@ -170,68 +194,72 @@ root.PointAssetForm = function() {
+        '<input type="checkbox" data-grouped-id="'+ asset.groupedId +'" class="form-control suggested-checkbox" name="' + asset.publicId + '" id="' + asset.publicId + (asset.groupedId ? ('-' + asset.groupedId) : '') + '"' + state + '>' +
+        '</div>';
+  };
+
   this.boxEvents = function (rootElement, selectedAsset, localizedTexts, authorizationPolicy, roadCollection, collection){};
 
   this.renderAssetFormElements = function(selectedAsset, localizedTexts, collection, authorizationPolicy) {
     var asset = selectedAsset.get();
+    var wrapper = $('<div class="wrapper">');
+    var formRootElement = $('<div class="form form-horizontal form-dark form-point-asset">');
 
-    if (selectedAsset.isNew()) {
-      return '' +
-        '<div class="wrapper">' +
-        '  <div class="form form-horizontal form-dark form-pointasset">' +
-        me.renderValueElement(asset, collection) +
-        suggestedAssetCheckBox(selectedAsset, authorizationPolicy) +
-        '  </div>' +
-        '</div>';
+    if (selectedAsset.isNew() && !selectedAsset.getWasOldAsset()) {
+      formRootElement = formRootElement.append(me.renderValueElement(asset, collection, authorizationPolicy));
     } else {
-      return '' +
-        '<div class="wrapper">' +
-        '  <div class="form form-horizontal form-dark form-pointasset">' +
-        me.renderFloatingNotification(asset.floating, localizedTexts) +
-        '    <div class="form-group">' +
-        '      <p class="form-control-static asset-log-info">Lis&auml;tty j&auml;rjestelm&auml;&auml;n: ' + me.informationLog(asset.createdAt, asset.createdBy) + '</p>' +
-        '    </div>' +
-        '    <div class="form-group">' +
-        '      <p class="form-control-static asset-log-info">Muokattu viimeksi: ' + me.informationLog(asset.modifiedAt, asset.modifiedBy) + '</p>' +
-        '    </div>' +
-        me.userInformationLog(authorizationPolicy, selectedAsset) +
-        me.renderValueElement(asset, collection, authorizationPolicy) +
-        suggestedAssetCheckBox(selectedAsset, authorizationPolicy) +
-        '    <div class="form-group form-group delete">' +
-        '      <div class="checkbox">' +
-        '        <input type="checkbox" id="delete-checkbox">' +
-        '      </div>' +
-        '      <p class="form-control-static">Poista</p>' +
-        '    </div>' +
-        '  </div>' +
-        '</div>';
+      var deleteCheckbox = $(''+
+          '    <div class="form-group form-group delete">' +
+          '      <div class="checkbox" >' +
+          '        <input id="delete-checkbox" type="checkbox">' +
+          '      </div>' +
+          '      <p class="form-control-static">Poista</p>' +
+          '    </div>' +
+          '  </div>' );
+      var logInfoGroup = $( '' +
+          '    <div class="form-group">' +
+          '      <p class="form-control-static asset-log-info">Lis&auml;tty j&auml;rjestelm&auml;&auml;n: ' + this.informationLog(asset.createdAt, asset.createdBy) + '</p>' +
```

Comment:
This part is not newly implemented code. It was already there from the start, I just encapsulated it in a jQuery variable instead of html in a string concatenation. I would rather not change it since it must have been used for a reason